examples/example-revenue-report/src/classes/Service/TemplateRenderer.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+class TemplateRenderer
+{
+    public function render(string $__template, array $__parameters): void
+    {
+        foreach ($__parameters as $__name => $__value) {
+            ${$__name} = $__value;
```

Comment:
what if $__name is not string?

Reply:
Added `@param array<string, mixed> $__parameters` annotation. I don't think we need to be too strict about it or at least not until PHP has generics.